doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -157,16 +157,22 @@ You must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option to supp
 An application can communicate with the Fast Pair subsystem using API calls and registered callbacks.
 The Fast Pair subsystem uses the registered callbacks to inform the application about the Fast Pair related events.
 
-The application must register the callbacks before it starts to operate as the Fast Pair Provider and advertise Bluetooth LE packets.
+The application must register the callbacks before it enables the Fast Pair subsystem and starts to operate as the Fast Pair Provider and advertise Bluetooth LE packets.
 To identify the callback registration functions in the Fast Pair API, look for the ``_register`` suffix.
 Set your application-specific callback functions in the callback structure that is the input parameter for the ``..._register`` API function.
 The callback structure must persist in the application memory (static declaration), as during the registration, the Fast Pair module stores only the memory pointer to it.
 
-The standard Fast Pair API (without extensions) currently supports the :c:func:`bt_fast_pair_info_cb` function (optional) for registering application callbacks.
+The standard Fast Pair API (without extensions) currently supports the :c:func:`bt_fast_pair_info_cb_register` function (optional) for registering application callbacks.
 
 The standard Fast Pair (without extensions) does not require registration of any callback type, meaning all callbacks are optional.
 
-Apart from the callback registration, no additional operations are needed to integrate the standard Fast Pair implementation.
+After the callback registration, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
+Before performing the :c:func:`bt_fast_pair_enable` operation, you must enable Bluetooth with the :c:func:`bt_enable` function and load Zephyr's :ref:`zephyr:settings_api` with the :c:func:`settings_load` function.
+The Fast Pair subsystem readiness can be checked with the :c:func:`bt_fast_pair_is_ready` function.
+The Fast Pair subsystem can be disabled with the :c:func:`bt_fast_pair_disable` function.
+In the Fast Pair subsystem disabled state, most of the Fast Pair APIs are not available.
```

Comment:
Seems like the second part of @kapi-no's comment was overlooked. Consider adding it as follows:

`The Fast Pair GATT Service is statically defined, so it's still present in GATT database after Fast Pair is disabled. In the Fast Pair subsystem disabled state, GATT operations on the Fast Pair service are rejected.